.github/labeler.yml

```diff
@@ -0,0 +1,47 @@
+translation:
+- po/**
+
+'translation/bn':
+- po/bn.po
+
+'translation/da':
+- po/da.po
+
+'translation/de':
+- po/de.po
+
+'translation/el':
+- po/el.po
```

Comment:
Nice idea! I'm worried about having to maintain the list of languages here, but I guess we can add them on a best-effort basis?

Reply:
Yes. [This action](https://github.com/actions/labeler#inputs) does not support programmatically adding labels. The configuration here falls back to `translation` label by default for new languages.

Reply:
Okay, sounds good. Could you add a line to `TRANSLATIONS.md` to ask people to update the configuration when adding a new translation?

Reply:
Done.

Reply:
Thank you!